pyquil/paulis.py

```diff
@@ -306,6 +306,9 @@ def __str__(self):
         out = "%s*%s" % (self.coefficient, '*'.join(term_strs))
         return out
 
+    def __repr__(self):
+        return str(self)
```

Comment:
I think you can get the same effect by just renaming the `__str__` method to `__repr__`.  If an object implements `__repr__` but not `__str__`, Python falls back to `__repr__`.

https://docs.python.org/3.6/reference/datamodel.html#object.__repr__

Reply:
@appleby nice! I'll make the change